### PR TITLE
Add io.github.revisto.drum-machine

### PR DESCRIPTION
**Pull request by Revisto**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. 

Drum Machine is a modern application for creating, playing, and managing drum patterns. Perfect for musicians, producers, and anyone interested in rhythm creation, this application provides a simple interface for drum pattern programming.

- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.revisto.DrumMachine/io.github.revisto.DrumMachine.json
Comment (hfiguiere) [inline io.github.revisto.DrumMachine/io.github.revisto.DrumMachine.json:1]: this file MUST be in the top-level directory as per the documentation.

File: io.github.revisto.DrumMachine/io.github.revisto.DrumMachine.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "id" : "io.github.revisto.DrumMachine",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
```
Comment (hfiguiere) [inline io.github.revisto.DrumMachine/io.github.revisto.DrumMachine.json:4]: '47' is the current one.

File: io.github.revisto.DrumMachine/io.github.revisto.DrumMachine.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "modules" : [
+        "python-dependencies.json",
+        {
+            "name" : "drum-machine",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/revisto/drum-machine.git"
```
Comment (hfiguiere) [inline io.github.revisto.DrumMachine/io.github.revisto.DrumMachine.json:35]: a specific commit and/or tag is required.

**Comment by hfiguiere** [comment]
bot, build io.github.revisto.DrumMachine

File: io.github.revisto.DrumMachine.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "id" : "io.github.revisto.DrumMachine",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "drum-machine",
+    "finish-args" : [
+        "--share=network",
```
Comment (hfiguiere) [inline io.github.revisto.DrumMachine.json:8]: does it really need network access?

File: io.github.revisto.DrumMachine.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+    "id" : "io.github.revisto.DrumMachine",
```
Comment (hfiguiere) [inline io.github.revisto.DrumMachine.json:2]: id doesn't match the github repository.

File: io.github.revisto.DrumMachine.json (hunk 3 above)
Comment (Revisto) [reply io.github.revisto.DrumMachine.json]: No, I removed that. It was just added by default from GNOME Builder template, but the app doesn't need any network access to function.

**Comment by bbhtt** [comment]
bot, build io.github.revisto.DrumMachine

**Comment by bbhtt** [comment]
bot, build io.github.revisto.drum-machine

**Pull request closed without merging**